automation/test.sh

```diff
@@ -27,6 +27,21 @@ readonly TEMPLATES_SERVER="gs://kubevirt-vm-images"
 readonly BAZEL_CACHE="${BAZEL_CACHE:-http://bazel-cache.kubevirt-prow.svc.cluster.local:8080/kubevirt.io/kubevirt}"
 
 
+# Skip if it's docs changes only
+env
+CI_GIT_BASE_REF=${PULL_BASE_REF:-main}
+CI_GIT_ALL_CHANGES=$(git diff --name-only $CI_GIT_BASE_REF $PULL_PULL_SHA)
+CI_GIT_SKIP_CHANGES=$(git diff --name-only $CI_GIT_BASE_REF $PULL_PULL_SHA | grep -E "^(docs/|.*/OWNERS|.*\.md)" || :)
+if [[ "$CI_GIT_ALL_CHANGES" == "$CI_GIT_SKIP_CHANGES" ]]; then
```

Comment:
maybe we can simplify to just check if the `diff | grep -v` is empty please ?

Reply:
reduces number of variables, simpler imho

diff=$(... | grep -v markdowns)
if diff == empty list; exit 0; fi

Reply:
Ok, fair!